### DISCUSSION TITLE
Canvas in Video Stream

### DISCUSSION THREAD

**Discussion by yogeshkaharmis** [ant-media/Ant-Media-Server#5334] [Plugins/Custom Extensions]
hello friends,
I want to use canvas tag in ant media sdk in android for video streaming.
any one knows about this.
please tell me.

**Comment by USAMAWIZARD** [comment]
Hi @yogeshkaharmis 
How are you , can you please clarify your requirements and what are you trying to archive are you using android SDK ?

**Reply by yogeshkaharmis** [reply]
hello @USAMAWIZARD
I am find, tell me about you?
yes i am using android SDK for mobile app.
but in my app i want to use canvas for showing app logo on streaming

**Comment by USAMAWIZARD** [comment]
can you check [this](https://developer.android.com/develop/ui/views/graphics/drawables#frame-animation) if it helps 
alternatively you can also add a logo on server side check [this](https://community.antmedia.io/topic/1124-add-watermark-using-plugin/) for adding logo on server side

**Reply by yogeshkaharmis** [reply]
ok.
